PSRabbitMq/Send-RabbitMqMessage.ps1

```diff
@@ -63,21 +102,51 @@
         [string]$Key,
 
         [Parameter(Mandatory=$true,ValueFromPipeline=$true)]
+        [Alias('Payload')]
         $InputObject,
 
         [switch]$Persistent,
 
         [Int32]$Depth = 2,
 
-        [PSCredential]$Credential,
+        [PSCredential]
+        [System.Management.Automation.Credential()]
+        $Credential,
 
         [System.Security.Authentication.SslProtocols]$Ssl,
 
-        [parameter(Mandatory = $false)]
-        [string]$vhost
+        [string]$vhost = '/',
+
+        [ValidateSet('application/clixml+xml','application/json','text/xml', 'text/plain')]
```

Comment:
What if we change this parameter name to SerializeAs and take "CliXml","Json","Xml","Text" and then later I can add an additional parameter set that takes ContentType and raw bytes for InputObject?